src/cromshell/status/command.py

```diff
@@ -53,7 +81,7 @@ def main(config, workflow_id):
     ):
         ret_val = 1
         log.display_logo(io_utils.dead_turtle)
-    elif workflow_status == "Running":
+    elif workflow_status == cromshellconfig.WorkflowStatuses.Running.value[0]:
```

Comment:
This is weird.  I feel like there must be a better solution but we don't have to find it right now.

Reply:
It feels like there should be a less awkward way of matching against the enum values.  We shouldn't have to decompose the list manually. Maybe we should overload the equality option in the enum to check all the list values so we don't have to do this awkward thing.